### PR TITLE
Added blue mlh ribbon

### LINKED ISSUE #117: Add MLH badge
https://mlh.io/brand-guidelines#trust-badge

Also fill in the navbar background so the MLH badge isn't just floating

### PR DESCRIPTION
**Pull request by StanfordLin**
:tickets: **Ticket(s)**: Closes #117 

---

## :construction_worker: Changes

Added the blue MLH ribbon to the homepage of the site. If the width is less than phablet (width of 900px), the ribbon will be hidden.

## :thought_balloon: Notes

Colour of the navbar/ribbon may change, depending on what the design team feels. The out of the box colours that are provided can be found here: https://mlh.io/brand-guidelines#trust-badge 

## :flashlight: Testing Instructions

You can resize the browser window (can test on different browser) to check if the ribbon is hiding properly.

Currently we are using blue but here are the other colours
screen shot 2018-10-06 at 10 33 03 pm
screen shot 2018-10-06 at 10 21 30 pm
screen shot 2018-10-06 at 10 22 00 pm

### CODE REVIEW

File: web/components/home/Home.js [hunk 1]
```diff
@@ -132,6 +132,9 @@ class Home extends React.Component {
   render() {
     return (
       <div className="homepage overflow-hidden flex jc-center ai-center dir-col">
+      <a className="mlh-trust-badge"  href="https://mlh.io/seasons/na-2019/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2019-season&utm_content=black" target="_blank">
```
Comment (chamkank) [inline web/components/home/Home.js]: Ideally we want to store our external links in `components/home/External.js`. You can find an example of it being used on [line 151](https://github.com/nwplus/nwhacks2019/blob/45867811c1a7e3c91118331723a3305901f07356/web/components/home/Home.js#L151).

File: web/components/home/Home.js [hunk 2]
```diff
@@ -132,6 +132,9 @@ class Home extends React.Component {
...
+      <img src="https://s3.amazonaws.com/logged-assets/trust-badge/2019/mlh-trust-badge-2019-blue.svg" alt="Major League Hacking 2019 Hackathon Season" style={{width: '100%' }} />
```
Comment (chamkank) [inline web/components/home/Home.js]: 1. Maybe you can save the badge to `/assets/` and then import and use it (so our webpage doesn't have to make a round trip to the s3 bucket).
2. We have a utility class for `width: 100%` called [`fill-width`](https://github.com/nwplus/nwhacks2019/blob/45867811c1a7e3c91118331723a3305901f07356/web/styles/utils.sass#L6). You can use it like `className="mlh-trust-badge fill-width"`

File: web/components/home/index.sass [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+  .mlh-trust-badge
+    display: block
+    max-width: 100px
+    min-width: 60px
+    position: absolute
+    right: 100px
+    top: 64px
+    width: 10%
+    z-index: 10000
+    @media screen and (max-width: $phablet)
```
Comment (chamkank) [inline web/components/home/index.sass]: We have a utility class for this media query called [`scale-hide-phablet`](https://github.com/nwplus/nwhacks2019/blob/45867811c1a7e3c91118331723a3305901f07356/web/styles/breakpoints.sass#L71).

**Comment by chamkank** [comment]
@sherryyx Do you want the transparency gone from the navbar & what's your color preference?

**Comment by sherryyx** [comment]
@chamkank @StanfordLin 
Thanks for looking into this! I think the blue ribbon looks just fine. I forgot to take the MLH banner into consideration when we designed the navbar, it's probably a good idea to remove the transparency then.

**Pull request merged**